Automattic/sensei
# Add module, outline styles

Apart from module & outline block styles and colors, this contains a lot of small style adjustment for various edge cases and combinations, generally aiming to support as much as of the theme's visuals as possible. 

### Changes proposed in this Pull Request

* Move border from Module to the parent Outline block — Looks like this was the design intention originally.
* Add module block styles (Default, Minimal, and an unstyled one for easy custom CSS)
* Add bordered/borderless outline block style
* Add outline & module colors
* Improve theme color support
* Adjust border colors in a few places to use text color
* Adjust lesson block style to design (whole block clickable), add a highlight for hover


### Testing instructions

* Create an outline with modules & lessons
* Customize appearance, check that everything looks the same on the frontend
* Customize colors using theme color palettes (the default colors in the Color Settings)
* On the frontend, Customize theme and check that changing theme colors applies to the outline colors too
* Check with a few themes


### Screenshot / Video

<img width="607" alt="image" src="https://user-images.githubusercontent.com/176949/94318130-1a4b5080-ff88-11ea-9a8f-7f55978ca953.png">
<img width="607" alt="image" src="https://user-images.githubusercontent.com/176949/94318019-e07a4a00-ff87-11ea-954e-0178b1586013.png">
